### ISSUE TITLE
Hyperlinks in descriptions

### ISSUE THREAD

**Issue by danidi** [wikipathways/meta-data-action#10]
https://new.wikipathways.org/pathways/WP4792.html contains two links in the description. On the old page, they were clickable, here it seems they are not correctly formatted to be a clickable link.

**Comment by AlexanderPico** [comment]
Good catch! @mkutmon Is this something we could translate as part of the metadata action? Basically, we need to detect (by grep) mediawiki markdown for links (e.g., `[https://link.com text]`) and translate to html (e.g., `<a href="https://link.com" target="_blank">text</a>`).

**Comment by Chris-Evelo** [comment]
This makes me think we could do CURIE (compact identifier) support as well? That would mean detecting a CURIE for what it is, like UNIPROT:P12345. And then use, ideally, a resolver like identifiers.org or nt2.net to go to the actual link (so we do not have to support resolving, just have a standard way to go to the resolver instead).

**Comment by AlexanderPico** [comment]
Great minds think alike :) Egon, Helena and I just implemented this for the "Participants" table on the new site. The code is all on place, but the CURIES just need to be fixed for the links to actually work properly. Should be working by next week. 

For example, the table includes `ncbigene:1234` and the new Jekyll site turns that into https://bioregistry.io/ncbigene:1234.

**Comment by khanspers** [comment]
It seems this issue is in the wrong repo (I originally moved it here), and should probably be in https://github.com/wikipathways/meta-data-action. Transferring.

**Comment by egonw** [comment]
Closing this issue. All links are now Markdown style and all work on the new website.
